packages/tictactoe/src/redux/game/reducer.ts

```diff
@@ -10,7 +10,7 @@ import { LoginSuccess, LOGIN_SUCCESS } from '../login/actions';
 
 import hexToBN from '../../utils/hexToBN';
 import bnToHex from '../../utils/bnToHex';
-import { RESTING } from 'src/core/positions';
+import { RESTING } from '../../core/positions';
```

Comment:
@lalexgap Curious, do you know why imports like `src/core/positions` do not work?